content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+Author: Fatema Boxwala
+Title: Constitution
+hidden: true
+
+# Structure of Organization #
+
+The Women in Computer Science Undergraduate Subcomittee is a
+subcommittee of Women in Computer Science, a standing committee of the
+Cheriton School of Computer Science. For the remainder of this document,
+this standing committee will be referred to as the "WiCS Committee". All
+references to: "the undergraduate committee", "the committee", and
+"committee members" refer to the Undergraduate Subcommittee unless
+otherwise specified.
+
+## Purpose and Vision ##
+
+The undergraduate committee aims to create a safe environment for cis
+and trans women as well as non-binary individuals in computer science,
+and to advocate for the inclusion of these marginalized groups in the
+computer science community at large.
```

Comment:
Taking a broader overview on this section, is our primary goal really just to make a safe environment for women? I'd like to think we're a bit loftier thinking than that... From the about page, "The Women in Computer Science Undergraduate Committee is a student-run initiative that supports the success of women in computer science at [the University of] Waterloo." I kind of prefer that.

The CoC says "A primary goal of the WiCS Undergraduate Committee is to be inclusive towards and supportive of women interested in computer science." Personally, I hate the phrasing "inclusive towards", but I couldn't think of anything better at the time.